components/Tabs/src/private/MDCItemBar.m

```diff
@@ -1,5 +1,6 @@
 /*
```

Comment:
Can you double check your setup? It looks like the style from material-components-ios/.clang-format isn't being applied - the mangling below seems to be mostly due to wrapping to 80 columns